.gitignore

```diff
@@ -12,7 +12,8 @@ typechain-types
 # cache
 *.dbg.json
 # artifacts
-
+artifacts
+cache
```

Comment:
artifacts and cache were intentionally added so that we could write test cases with hardhat in the sdk